fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCParameterBuildingVisitor.java

```diff
@@ -558,10 +566,10 @@ public boolean visit(java.lang.String elementName, int elementIndex, Timing timi
 
     @Override
     public boolean visit(java.lang.String elementName, int elementIndex, Location.Position position) {
-        Parameter p = new Parameter();
+        LocationParmVal p = new LocationParmVal();
         p.setName(searchParamCode);
         
-        // The following code ensures that the lat/lon is only added when there is a pair. 
+        // The following code ensures that the lat/long is only added when there is a pair. 
```

Comment:
```suggestion
        // The following code ensures that the lat/lon is only added when there is a pair. 
```

Reply:
this comment refers to the lat/lon that follow. 
long is a reserved word, so I wanted to avoid it in this comment

Reply:
reverted